plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergMaterializedViewTest.java

```diff
@@ -411,6 +411,39 @@ public void testDetectStaleness()
         assertUpdate("DROP MATERIALIZED VIEW materialized_view_join_part_stale");
     }
 
+    @Test
+    public void testMaterializedViewOnExpiredTable()
+    {
+        Session sessionWithShortRetentionUnlocked = Session.builder(getSession())
+                .setCatalogSessionProperty("iceberg", "expire_snapshots_min_retention", "0s")
+                .build();
+
+        assertUpdate("CREATE TABLE mv_on_expired_base_table AS SELECT 10 a", 1);
+        assertUpdate("""
+                CREATE MATERIALIZED VIEW mv_on_expired_the_mv
```

Comment:
`mv_on_expired_the_mv` ... is there some logic behind this name?

Reply:
`mv_on_expired_` -- relates to the test method name
`base_table`, `the_mv` -- identifies relation within the test